test/unit/spec/localparticipant.js

```diff
@@ -544,21 +494,21 @@ describe('LocalParticipant', () => {
                   ? localTrack : (localTrack.mediaStreamTrack || localTrack._trackSender));
 
               test.signaling.tracks.set(localTrack.id, makeTrackSignaling(localTrack.id, 'foo'));
-              test.participant.tracks.set(localTrack.id, track);
-              test.participant[`${kind}Tracks`].set(localTrack.id, track);
-              test.participant.trackPublications.set('foo', { track, sid: 'foo' });
-              test.participant[`${kind}TrackPublications`].set('foo', { track, sid: 'foo' });
+              test.participant._tracks.set(localTrack.id, track);
+              test.participant[`_${kind}Tracks`].set(localTrack.id, track);
+              test.participant.tracks.set('foo', { track, sid: 'foo' });
+              test.participant[`${kind}Tracks`].set('foo', { track, sid: 'foo' });
```

Comment:
I don't really like these kind of testing where we set private APIs (or mutate APIs we shouldn't, like `tracks`) but we were doing this before, so it's fine for now. One thing we may want to consider in 2.x—changing the Map types we expose in our public API to be some kind of ReadOnlyMap or something that forbids `set`/`delete`/`clear`.